src/components/App/App.test.js

```diff
@@ -44,10 +45,13 @@ describe('<App />', () => {
     initApp: jest.fn()
   };
   const mockStore = configureStore();
-  const store = mockStore({ application: initialState });
+  const store = mockStore({
+    application: initialApplicationState,
+    theme: initialThemeState
```

Comment:
Is there any specific reason for theme to live outside application?

Reply:
As we now don't have a theme component anymore, not really. I moved all theme-related redux code into application now, i.e. including the selectors/reducers/actions. The only issue is that the only place where the toggleDarkMode() action is used, on the user settings page, all application actions are imported. I don't know if that's an issue though (if it gets optimized away or if it would be better to only import one action or move the action to a different file).